config/marconi.mk

```diff
@@ -5,7 +5,7 @@ INCLUDE += -I$(NETCDF_INC) -I$(HDF5_INC)
 GLFLAGS  = -lm
 CC=icc
 MPICC=mpiicc
-OPT=-O3 -xHost
+OPT=-O3 -xHost -fma -xMIC-AVX512 -finline-functions -align
```

Comment:
Nice! From our meeting I remember that -fma, -finline-functions and -align are not specific for the KNL architecture and can also be used in other config files with intel compilers? What about -xMIX-AVX512? Should we also use this flag on the A1 partition?